release/staging-resources-cdk/README.md

```diff
@@ -53,6 +53,17 @@ The following CDK commands all require defining context. The context variables a
 * `archivesBucketName` - The name of the S3 bucket you will use to deploy. This bucket must be in the same region as your stack.
 * `dataPrepperOrganization` - The name of the GitHub organization which has the `data-prepper` repository. This allows you to create staging environments for forks. The default value is `opensearch-project`.
 
+The following command will deploy the CDK stack and create a new S3 bucket. If you'd like to use an existing S3 bucket, see the section below
+for deploying individual stacks.
+
+```
+cdk deploy --all --context archivesBucketName={s3-bucket-name} --context dataPrepperOrganization={data-prepper-organization-name}
```

Comment:
QUES: is there any reason this does not work for existing S3 bucket? The only diff with section below I can find is `--all` flag.

Reply:
Deploying the whole stack will create a new S3 bucket. I'm sure we could improve this workflow if somebody needed something smoother. But, for now, you can create it the default way easily, or with a custom bucket using some additional effort.